README.md

```diff
@@ -122,6 +122,17 @@ preview.show(fileId, accessToken, {
 | showDownload | false | Whether download button is shown |
 | useHotkeys | true | Whether hotkeys (keyboard shortcuts) are enabled |
 
+Viewer specific options:
+```javascript
+preview.show(fileId, accessToken, {
+    ...,
+    viewers: { Document: { useFindBar: false } }
+});
+```
+| Option | Default | Description |
+| --- | --- | --- |
+| useFindBar | true | Setting to false will enable the browser's findBar in place of the viewers. Applies to `Document`, `Presentation` viewers. |
```

Comment:
Please move this to the document specific readme https://github.com/box/box-content-preview/blob/master/src/lib/viewers/doc/README.md.